supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -32,29 +32,29 @@ final case class TesTask(jobDescriptor: BackendJobDescriptor,
                          jobShell: String,
                          outputMode: OutputMode) {
 
-  private val workflowDescriptor = jobDescriptor.workflowDescriptor
-  private val workflowName = workflowDescriptor.callable.name
-  private val fullyQualifiedTaskName = jobDescriptor.taskCall.fullyQualifiedName
-  private val workflowExecutionIdentityConfig: Option[WorkflowExecutionIdentityConfig] =
+  private lazy val workflowDescriptor = jobDescriptor.workflowDescriptor
+  private lazy val workflowName = workflowDescriptor.callable.name
+  private lazy val fullyQualifiedTaskName = jobDescriptor.taskCall.fullyQualifiedName
+  private lazy val workflowExecutionIdentityConfig: Option[WorkflowExecutionIdentityConfig] =
```

Comment:
All these vals were made lazy so that `TesTask` could be effectively tested with mocks, without needing to mock out a bunch of stuff unnecessary to any particular test.